<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="UIfiles.qrc">
    <normaloff>:/icons/Icons/calculate_black_24dp.svg</normaloff>:/icons/Icons/calculate_black_24dp.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: white;
	background-color: #202020;
	font-family: &quot;Segoe UI&quot;;
	font-size: 16pt;
	font-weight: 500;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>40</width>
    <height>40</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QLabel" name="MemoryReg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>75</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>11</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 11pt;
color: #909090;
margin-right: 7px;</string>
      </property>
      <property name="text">
       <string>55555 =</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="EnterReg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>38</pointsize>
        <weight>62</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 38pt;
border: none;
margin-right: 7px;
</string>
      </property>
      <property name="maxLength">
       <number>16</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_opposite">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 11pt;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>1/ùë•</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_persent">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 12pt;
	font-weight: 400;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btn_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_sign">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="btn_sub">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="UIfiles.qrc">
          <normaloff>:/icons/Icons/remove_white_24dp.svg</normaloff>:/icons/Icons/remove_white_24dp.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_square">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 11pt;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>ùë•¬≤</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btn_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btn_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="btn_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="btn_point">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>,</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btn_0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_c">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 11pt;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_sqrt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 11pt;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>‚àöùë•</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btn_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="UIfiles.qrc">
          <normaloff>:/icons/Icons/add_white_24dp.svg</normaloff>:/icons/Icons/add_white_24dp.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btn_multiply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="UIfiles.qrc">
          <normaloff>:/icons/Icons/close_white_24dp.svg</normaloff>:/icons/Icons/close_white_24dp.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_ce">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 11pt;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>CE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="btn_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="btn_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btn_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="btn_divide">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>√∑</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_backspace">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	background-color: #323232;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #3b3b3b;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="UIfiles.qrc">
          <normaloff>:/icons/Icons/backspace_white_24dp.svg</normaloff>:/icons/Icons/backspace_white_24dp.svg</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="btn_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 13pt;
	background-color: #3b3b3b;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #323232;
}

QPushButton:pressed {
	background-color: #2f2f2f;
}</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="btn_result">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color: black;
	font-family: &quot;Segoe Ui Light&quot;;
	font-size: 16pt;
	background-color: #76b9ed;
	border: none;
	border-radius: 5px;
}

QPushButton:hover {
	background-color: #6da9d8;
}

QPushButton:pressed {
	background-color: #649ac3;
}</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="UIfiles.qrc"/>
 </resources>
 <connections/>
</ui>
